src/doppler_env/__init__.py

```diff
@@ -1,26 +1,52 @@
 """
 Imported by doppler_env.pth Automatically inject Doppler secrets as environment variables when spawning a Python process if the `DOPPLER_ENV` environment variable is set
 """
+from base64 import b64encode
+import io
 import os
+import subprocess
+import sys
+import urllib.request
+import urllib.error
 
+from dotenv import load_dotenv
 
-def doppler_secrets_env():
-    import io
-    import subprocess
-    from dotenv import load_dotenv
+def print_debug_info():
+    print(f'Python path: {sys.executable}')
+    print(f'Working directory: {os.getcwd()}')
 
+def fetch_cli():
     DOPPLER_ENV_COMMAND = os.environ.get(
         'DOPPLER_ENV_COMMAND', 'doppler secrets download --no-file --format env'
     )
 
     # If non-zero exit code, catch Python exception so only output is stderr from Doppler CLI
     try:
-        env_vars = subprocess.check_output(DOPPLER_ENV_COMMAND.split()).decode('utf-8')
+        return subprocess.check_output(DOPPLER_ENV_COMMAND.split()).decode('utf-8')
     except subprocess.CalledProcessError as err:
-        exit(err.returncode)
+        print_debug_info()
```

Comment:
Should we still exit here?

Reply:
No, that was a mistake on my part in the original implementation as exiting prevents their application from starting which is a bit aggresive.